doc/source/atomgroup.rst

```diff
@@ -19,10 +19,12 @@ Atom selection language
 
 AtomGroup instances are typically created with :meth:`Universe.select_atoms <MDAnalysis.core.universe.Universe.select_atoms>` or by manipulating another :class:`~MDAnalysis.core.groups.AtomGroup`, e.g. by slicing.
 
-.. code-block:: python
+.. ipython:: python
```

Comment:
What does an `ipython::` block do? Just curious.

Reply:
Evaluate?

That's cool!

Reply:
Yes, `ipython:: python` lets you [write Python input](https://ipython.readthedocs.io/en/stable/sphinxext.html#writing-pure-python-code) and IPython will handle the output, generating errors during doc compilation if there are any. Good for handling typos and not having to think about output.